src/definitions.jl

```diff
@@ -0,0 +1,74 @@
+
+using LightGraphs, MetaGraphs
+
+import JSON
+
+# TODO: Create functions for each
+# Cause - is concept label, Effect - definition label
+# within a RelationMention with labels "Definition, Entity"
+
+definitions_output_file = "/Users/scott/Documents/JuliaProjects/SemanticModels.jl/test/definitions.jl/automates_output_1sentence_definition.json"
+def_output_dir = "/Users/scott/Documents/JuliaProjects/SemanticModels.jl/test/definitions.jl"
+
+
+function info_extract_from_automates(output_dir::String)
+    file_list = readdir(output_dir)
+    graph = DiGraph()
+    changes_recorded = []
+    v_count = 0
+    for def_file in file_list
+        json_res = JSON.parsefile(string(def_output_dir, "/",def_file))
+        # json_res = JSON.parsefile(definitions_output_file)
+        mentions = json_res["mentions"]
+        # println("JSON Mentions is:\n")
+        # println(mentions)
+        # println("End mentions\n")
+        for val in mentions
+            if haskey(val,"type") && val["type"] == "RelationMention"
+                JSON.print(val, 4)
+                v_count += 1
+                cause = val["arguments"]["cause"][1]["text"]
+                effect = val["arguments"]["effect"][1]["text"]
+                println(string("Cause is: ", cause, "\n"))
```

Comment:
A shorthand for this is `println("Cause is: $cause")`

The `$var` inside a string expand the variable name into its value you can execute arbitrary code in the string formatting like this `string ="f(x)=$(f(x))` which will call `f(x)` on the right hand side.